client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/RequestStats.tsx

```diff
@@ -1,49 +1,32 @@
 import React from 'react';
-import { Box } from '@openmsupply-client/common';
+import { Box, Paper } from '@openmsupply-client/common';
 import { DraftRequestLine } from '../hooks';
-import { StockDistribution } from './StockDistribution';
 import { ConsumptionHistory } from './ConsumptionHistory';
-import { StockEvolution } from './StockEvolution';
 
 export interface RequestStatsProps {
   draft?: DraftRequestLine | null;
 }
 
 export const RequestStats = ({ draft }: RequestStatsProps) => {
   return (
-    <Box
+    <Paper
       sx={{
+        mt: 2,
+        p: 2,
         minHeight: 200,
         maxHeight: 500,
-        width: 500,
+        width: '100%',
+        overflow: 'auto',
       }}
     >
-      {!!draft && (
-        <StockDistribution
-          availableStockOnHand={draft?.itemStats?.availableStockOnHand}
-          averageMonthlyConsumption={
-            draft?.itemStats?.averageMonthlyConsumption
-          }
-          suggestedQuantity={draft?.suggestedQuantity}
-        />
-      )}
       <Box
         display="flex"
         sx={{ paddingLeft: 2, paddingRight: 2 }}
         flexDirection="column"
         justifyContent="space-between"
       >
-        {draft?.isCreated ? (
-          <Box display="flex" height={289} />
-        ) : (
-          <>
-            <Box paddingBottom={2}>
-              <ConsumptionHistory id={draft?.id || ''} />
-            </Box>
-            <StockEvolution id={draft?.id || ''} />
```

Comment:
New designs, doesn't point into how we should incorporate values from `StockEvolution`. Excluded for now unless we think otherwise